x-pack/plugin/eql/qa/common/src/main/resources/test_queries.toml

```diff
@@ -2,6 +2,15 @@
 query = 'process where serial_event_id = 1'
 expected_event_ids  = [1]
 
+[[queries]]
+query = 'process where string(serial_event_id) = "1"'
+expected_event_ids  = [1]
+
+[[queries]]
+note = "check that string(null) returns 'null'"
+expected_event_ids  = [1, 2]
+query = 'process where opcode == 3 and string(ppid) == "null"'
+
```

Comment:
The original idea was to keep this file unchanged from the original eql implementation 
https://github.com/endgameinc/eql/blob/master/eql/etc/test_queries.toml
so we can sync up easier with the reference repo/impl

and add more queries into separate file(s), could be just one, could be individual files for specific extensions 
started doing something like this in my other PR
https://github.com/elastic/elasticsearch/pull/54277/files/be0b67cdf5022f38ec3610a68bbb91af8b10b8b3#diff-00cfec987b028a9cc67295e57c02cf68R1